src/HCSR04.h

```diff
@@ -13,25 +13,32 @@ class UltraSonicDistanceSensor {
     /**
      * @param triggerPin  Digital pin that is used for controlling sensor (output).
      * @param echoPin  Digital pin that is used to get information from sensor (input).
+     * @param maxDistanceCm  Maximum distance sensor can measure, defaults to 4m for HC-SR04.
+     *                       Set to maximum measuring distance to reduce measurement time.
```

Comment:
You might want to set this value if you are using different sensor than HC-SR04 or if you don't care about distances larger than whatever you will set it to (therefore reducing time it takes for a single measurement).
```suggestion
     *                       You might want to set this value if you are using different sensor than HC-SR04
     *                       or if you don't care about distances larger than whatever you will set it to
     *                       (therefore reducing time it takes for a single measurement).
```